config.yaml

```diff
@@ -18,15 +18,24 @@ timeline:
 # Cameras and other input data sources
 # Using Home Assistant conventions to ease upcoming integration
 sources:
-#  front_door_camera:
```

Comment:
OK, we can switch default to a local camera. Almost all dev computers nowadays have one. The question is whether all of them will work with /dev/video0 or we need a higher level abstraction like uri:default_local_camera and then parse that into something specific to the current environment. That would be my expectation if I was a new user. Usually apps don't ask the user how to local the default camera. They work with the hosting OS to do so.

Reply:
Yup, this is a good idea indeed. Need to focus a bit more on OS detection in that case if gst does not have a "trick" for that. Anyway would be better to test on win and mac first, see also my comment above